src/panels/lovelace/cards/hui-weather-forecast-card.ts

```diff
@@ -87,11 +87,13 @@ class HuiWeatherForecastCard extends LitElement implements LovelaceCard {
   }
 
   public getCardSize(): number {
+    const [weather, forecast] = this._prepareRenderData();
```

Comment:
`getCardSize` should only use config, and not state info

Reply:
Then this PR has no purpose as an alignment would require state and the existing behavior is the only solution.